tests/test_unittest/test_gtscript_frontend.py

```diff
@@ -476,3 +476,44 @@ def func(in_field: gtscript.Field[np.float_], out_field: gtscript.Field[np.float
                 with computation(PARALLEL), interval(...):
                     tmp[0, 0, 0] = 2 * in_field
                     out_field = tmp
+
+
+class TestRegion:
+    def test_region_parsing(self):
```

Comment:
Maybe name `test_parsing` since this is already on `TestRegion`.

Reply:
Renamed!